reactive/load_balancer.py

```diff
@@ -168,7 +162,21 @@ def install_load_balancer():
 
         port = hookenv.config('port')
         hookenv.open_port(port)
-        services = apiserver.services()
+        services = []
+        if apiserver:
+            services.extend(apiserver.services())
+        for request in lb_consumers.all_requests:
+            services.append({
+                'hosts': [
+                    {
+                        'hostname': address,
+                        'port': port,
+                    }
+                    for address, port in itertools.product(
+                        request.backends, request.port_mapping.values()
```

Comment:
Hrm, yeah, this is a case where the philosophy diverges, since the `loadbalancer` interface (and all other LB providers) expect the consumer to dictate what the ingress port should be. This should honor the port on the request and the config option on the kapi-lb charm should be deprecated.